public/views/index-template.html

```diff
@@ -199,9 +199,10 @@
         </p>
         <p>
           1. This could be caused by your reverse proxy settings.<br /><br />
-          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath<br />
+          2. Otherwise, if you host grafana under subpath make sure your grafana.ini root_url setting includes subpath
```

Comment:
Otherwise to what? Using a reverse proxy?

Reply:
@papagian what about this?

```suggestion
          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath. If not using a reverse proxy make sure to set serve_from_sub_path to true.
```